### ISSUE TITLE
Cohort table listed in CDM V.5.3 

### ISSUE THREAD

**Issue by solmazeradat** [OHDSI/WhiteRabbit#358]
Hi,

I had a question about Rabbit In A Hat. I noticed that the table **Cohort** under vocabulary tables is listed when selecting V5.3 (See screenshot below) 

![image](https://user-images.githubusercontent.com/11243960/199746257-bbdf7f58-ec8d-4361-b6bb-f9ca229b4759.png)

From the documentation [cdmV5.3](https://ohdsi.github.io/CommonDataModel/cdm53.html#Vocabulary_Tables) this table is not listed, however, does appear in V5.4. 

Many thanks,
Solmaz

**Comment by don-torok** [comment]
Defining a cohort is typically an operation that is done after the ETL is complete. Do you have a 'use case' where the cohort is filled from the source data?

**Comment by solmazeradat** [comment]
My question was more around why the table Cohort appears in the RabbitInAHat tool when setting the target database to V5.3? From reading the documentation it looks like the Cohort table was added in V.5.4.

**Comment by MaximMoinat** [comment]
@solmazeradat Indeed, this incosistent, only cohort_definition should exist in v5.3.

That said, the cohort (or cohort_definition) table is typically not populated during ETL. Does the presence of the cohort table in RiaH break a workflow for you?

**Comment by solmazeradat** [comment]
Hi @MaximMoinat - no it does not break anything for us.
